letsencrypt/auth_handler.py

```diff
@@ -139,11 +154,13 @@ def _respond(self, cont_resp, dv_resp, best_effort):
 
         # Check for updated status...
         try:
-            self._poll_challenges(chall_update, best_effort)
+            result = self._poll_challenges(chall_update, best_effort)
         finally:
             # This removes challenges from self.dv_c and self.cont_c
             self._cleanup_challenges(active_achalls)
 
+        return result
```

Comment:
I'd update the doc string to mention that you've added a return value to this function